commands/test/test-command.ts

```diff
@@ -417,3 +423,4 @@ export const execute = async (interaction: any) => {
     sendQuestion(interaction);
 };
 
+export { specificQuestionMessage }
```

Comment:
btw use individual exports instead of putting all exports at the end of the file